### PR TITLE
Added test to run Libunwind tests

### PR DESCRIPTION
**Pull request by poojabs7**
That executes self tests of unwind library

Signed-off-by: Pooja <pooja@linux.vnet.ibm.com>

### CODE REVIEW

File: perf/libunwind.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        dist = distro.detect()
+        smm = SoftwareManager()
+        deps = ['gcc', 'libtool', 'autoconf', 'automake', 'make']
+        if dist.name == 'Ubuntu':
+            deps.extend(['dh-autoreconf', 'dh-dist-zilla', 'g++',
+                         'texlive-extra-utils'])
+        elif dist.name in ['SuSE', 'rhel', 'fedora', 'redhat']:
+            deps.extend(['gcc-c++'])
+        else:
+            self.fail('No packages list for your distro.')
```
Comment (PraveenPenguin) [inline perf/libunwind.py]: i think it can be `test not supported in distro ` as well please use `self.cancel `

File: perf/libunwind.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+        for package in deps:
+            if not smm.check_installed(package) and not smm.install(package):
+                self.error("Failed to install %s, which is needed for"
```
Comment (PraveenPenguin) [inline perf/libunwind.py]: here also please use `self.cancel`

File: perf/libunwind.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        tarball = self.fetch_asset('vanilla_pathscale.zip', locations=[
+            'https://github.com/pathscale/libunwind/archive/'
+            'vanilla_pathscale.zip'], expire='7d')
+        archive.extract(tarball, self.srcdir)
+        self.srcdir = os.path.join(self.srcdir, 'libunwind-vanilla_pathscale')
+        os.chdir(self.srcdir)
+        process.run('./autogen.sh', shell=True)
+        '''
+        For configure options on different architecture please refer
+        https://github.com/pathscale/libunwind
```
Comment (PraveenPenguin) [inline perf/libunwind.py]: nice

**Comment by poojabs7** [comment]
sent new PR.
https://github.com/avocado-framework-tests/avocado-misc-tests/pull/581

**Pull request closed without merging**